code-challenges.rb

```diff
@@ -3,31 +3,115 @@
 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
 
-num1 = 7
+# num1 = 7
 # Expected output: '7 is odd'
-num2 = 42
+# num2 = 42
 # Expected output: '42 is even'
-num3 = 221
-# Expected output: '221 is odd'
+# num3 = 221
+# # Expected output: '221 is odd'
+
+newArray = [7, 42, 221]
+newNum = newArray.map do |value|
+    if value % 2 == 0
+        p "#{value} is even"
+    else
+        p "#{value} is odd"
+    end
+end
+
+# another way to do this task (down below)
+    num1 = 7
+        if num1 % 2 == 0
+            p "#{num1} is even"
+        else
+            p "#{num1} is odd"
+        end
+        
+    num2 = 42
+    if num2 % 2 == 0
+        p "#{num2} is even"
+    else
+        p "#{num2} is odd"
+    end
+
+    num3 = 221
+    if num3 % 2 == 0
+        p "#{num3} is even"
+    else
+        p "#{num3} is odd"
+    end
```

Comment:
Your code looks great. But the goal is to take the inputs as they are given and create a method that will handle all three.